Isosurfacing_3/include/CGAL/Default_gradients.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2022 INRIA Sophia-Antipolis (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Julian Stahl
+
```

Comment:
The file name does not correspond to the class names in the file.

Reply:
Should I separate the classes into different files then?

Reply:
Rename it just to `Gradients.h`   Which is maybe a too generic name in a directory `CGAL/`.